tests/enum/tenum.nim

```diff
@@ -6,7 +6,7 @@ ABCDC
 foo
 first0second32third64
 my value A1my value Bconc2valueCabc4abc
-my value A0my value Bconc1valueCabc3valueC
```

Comment:
Why did this change?

Reply:
the test was wrong, and was affected by #17162 ; x should be abc, not valueC